src/components/Navbar/Navbar.tsx

```diff
@@ -26,14 +29,52 @@ const Navbar = () => {
     fetchUser();
   }, [user, pathname]);
 
+  const handleLogout = () => {
+    logout();
+    window.location.reload();
+  };
+
+  const items: MenuProps['items'] = [
+    {
+      key: '1',
+      label: (
+        <a href="/add/post">
```

Comment:
Can we use `<Link>` from `react-router-dom`

Reply:
Yes